docs/configuration.rst

```diff
@@ -159,18 +159,25 @@ To create it manually, follow the specific deployment method documentation.
 
 To have the account created automatically, you just need to define a few environment variables:
 
-.. code-block:: bash
+- ``INITIAL_ADMIN_ACCOUNT``: the admin username: The first part of the e-mail address before the @.
+- ``INITIAL_ADMIN_DOMAIN``: the domain appendix: Most probably identical to the ``DOMAIN`` variable.
+- ``INITIAL_ADMIN_PW``: the admin password.
+- ``INITIAL_ADMIN_MODE``: one of the options below how the code should behave when it will try to create the admin user:
+  
+  - ``create``: (default) Will try to create user and will raise an exception if present.
+  - ``ifmissing``: if user exists, nothing happens, else it will be created.
+  - ``update``: user is created or, if it exists, its password gets updated.
+
+You should set ``INITIAL_ADMIN_MODE`` also to either ``update`` or ``ifmissing``. Leaving it with the default value could cause errors when restarting the system.  
```

Comment:
Bad wording `also to either`. I recommend to completely change line 171 to:
It is recommended to set ``INITIAL_ADMIN_MODE`` to either ``update`` or ``ifmissing``. Leaving it with the default value will cause an error when the system is restarted.

Reply:
Thanks for your feedback. I've improved this in an updated commit.